tpm2/test/tpm2_test.go

```diff
@@ -1102,6 +1102,38 @@ func TestReadPCR(t *testing.T) {
 	}
 }
 
+func TestPCRReset(t *testing.T) {
+	rw := openTPM(t)
+	defer rw.Close()
+	allZeroBytes := make([]byte, 32)
+
+	var fakeHashSum [32]byte
+	err := PCRExtend(rw, 16, AlgSHA256, fakeHashSum[:], "")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	pcrVal, err := ReadPCR(rw, 16 /*pcr*/, AlgSHA256)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if reflect.DeepEqual(allZeroBytes, pcrVal) {
```

Comment:
Here and elsewhere, I think this can just be bytes.Equal, they are just byte slices after all

Reply:
Right. Done.